documentation/blog/2025-04-23-things-need-to-know/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "4 Things You Need to Know Before Using Goose"
+description: "Learn what you need to get starte with Goose,local, open-source, and powered by the LLM of choice."
+authors: 
+    - ebony
+---
+![blog cover](cover.png)
+
+# 4 Things You *Actually* Need to Know Before Using Goose
+
+So you’ve heard about Goose. Maybe you saw a livestream, maybe someone on your team mentioned it, or maybe you just stumbled into our corner of the internet while trying to automate your dev setup. Either way—love that for you.
+
+<!-- truncate -->
+Goose is a local, open source AI agent that can automate tasks, interact with your codebase, and connect to a growing ecosystem of tools. But before you hit install, here are four things you should know to get the most out of it.
+
+---
+
+## So Wait—What *Is* Goose, Actually?
+
+Goose is an **MCP client**.
+
+That means it connects to tools and data through something called the **Model Context Protocol (MCP)**—an open standard that makes it possible for AI agents to interact with external systems through natural language. If you’ve used Claude Desktop, or Cursor you’ve already used an MCP client, even if you didn’t realize it.
```

Comment:
love it!

link to model context protocol here. there's still people that dont know what it is.

Reply:
maybe add Windsurf and Agent mode in VS Code as other clients